logging/logger_test.go

```diff
@@ -19,40 +43,141 @@ func TestSetLogLevel(t *testing.T) {
 	}
 }
 
+func prepareLogger(w io.Writer, v level) func() {
+	o := lgr
+	lgr = log.New(w, "", 0)
+	SetLogLevel(v)
+	lgr.SetFlags(0)
```

Comment:
SetLogLevel sets `lgr.SetFlags(log.LstdFlags)`.

`log.LstdFlags` will append current date and time at the front of the log, but that behavior makes tests difficult.